bin/sgcloud

```diff
@@ -14,3 +14,7 @@ Object.keys(actions).forEach(a => new actions[a]().register(commander))
 
 // activate argument parsing
 commander.parse(process.argv)
+
+// undefined if no cmd is given on cli, object if a cmd was found, string if not
+const executedCommand = commander.args.pop()
+if (!executedCommand || typeof executedCommand === 'string') commander.help()
```

Comment:
If `executedCommand` (if the user entered a command) it should show something like `Unknown command`.
To show the developer that he made a misstake